README.md

```diff
@@ -475,6 +475,7 @@ _[SIP](https://en.wikipedia.org/wiki/Session_Initiation_Protocol)/[IPBX](https:/
 - [PPnet](https://github.com/pixelpark/ppnet) - Create and host your own social network. `MIT` `Javascript`
 - [Pump.io](http://pump.io/) - Stream server that does most of what people really want from a social network. ([Source Code](https://github.com/e14n/pump.io)) `Apache-2.0` `Nodejs`
 - [remark42](https://remark42.com/) - A lightweight and simple comment engine, which doesn't spy on users. It can be embedded into blogs, articles or any other place where readers add comments. ([Demo](https://remark42.com/demo/), [Source Code](https://github.com/umputun/remark)) `MIT` `Go`
+- [rss2twtxt](https://github.com/prologic/rss2twtxt) -- An RSS/Atom feed aggregator command-line tool and web service that consume RSS/ATOM and produces [twtxt](https://twtxt.readthedocs.org) feeds for consumption by twtxt clients. ([Demo](https://feeds.twtxt.net)) `MIT` `Go`
```

Comment:
- Only a single dash between the main link and description please
- No inline links in description please, the syntax check/parser doesn't play well with them
- I'd rather have this as a sub-entry (indented three spaces) of twtxt.net

Reply:
Done! Hopefully this is _right_.